Doc/library/subprocess.rst

```diff
@@ -512,6 +512,27 @@ functions.
    .. versionchanged:: 3.2
       *start_new_session* was added.
 
+   If *groups* not null, the setregid() system call will be made in the child
+   process prior to the execution of the subprocess. If the provided value is
+   a string, the provided group name will be looked up via
+   :func:`grp.getgrgid()` and the value in `gr_gid` will be used. If *groups*
+   is a list, strings will be looked up via :func:`grp.getgrgid()` and the
+   values in `gr_gid` will be used. Integer values will be passed verbatim.
+   The first item in the list will be passed go setregid(), and the following
+   items will be passed to the setgroups() system call. On sytems without
+   setgroups(), the remaining items will be discarded.  (POSIX only)
+
+   .. versionchanged:: 3.8
+      *groups* was added.
```

Comment:
add groups=None and user=None to the class Popen signature on line 336 above.

Reply:
OK